Formula/itk.rb

```diff
@@ -41,30 +42,32 @@ class Itk < Formula
   fails_with gcc: "5"
 
   def install
-    args = std_cmake_args + %W[
+    # Avoid CMake trying to find GoogleTest even though tests are disabled
+    (buildpath/"Modules/ThirdParty/GoogleTest").rmtree
+
+    args = %W[
       -DBUILD_SHARED_LIBS=ON
-      -DBUILD_TESTING=OFF
       -DCMAKE_INSTALL_RPATH:STRING=#{lib}
       -DCMAKE_INSTALL_NAME_DIR:STRING=#{lib}
-      -DITK_USE_64BITS_IDS=ON
-      -DITK_USE_STRICT_CONCEPT_CHECKING=ON
```

Comment:
Dropped `BUILD_TESTING` as it is part of `std_cmake_args`.

Dropped `ITK_USE_STRICT_CONCEPT_CHECKING` as no longer used https://github.com/InsightSoftwareConsortium/ITK/blob/v5.2.1/CMakeLists.txt#L485-L487
```cmake
if(ITK_USE_STRICT_CONCEPT_CHECKING)
  message(WARNING "Compilers requiring ITK_USE_STRICT_CONCEPT_CHECKING are no longer suppported.  This variable is no longer used.")
endif()
```

Other changes are mainly re-ordering.